content/applications/finance/accounting/reporting/overview/customize.rst

```diff
@@ -1,90 +1,234 @@
+  a group-by value, this will correspond to the number of distinct grouping keys in the matched move
+  lines. Else, it will be the number of matched move lines.
 
--  **Is growth good when positive** : Used when computing the comparison
-   column. Check if growth is good (displayed in green) or not.
+You can also put a `-` sign at the beginning of the sub-formula to **invert** the sign of the
+result.
 
--  **Special date changer** : If a specific line in a report should not use
-   the same dates as the rest of the report.
+'Tax Tags' engine
+~~~~~~~~~~~~~~~~~
 
--  **Show domain** : How the domain of a line is displayed. Can be foldable
-   (``default``, hidden at the start but can be unfolded), ``always``
-   (always displayed) or ``never`` (never shown).
+A formula made for this engine consists of a name used to match tax tags. If such tags do not exist
+when creating the expression, they will be created.
 
-.. seealso::
-    * :doc:`main_reports`
+.. example::
+   If the formula is **tag_name**, the engine matches tax tags **+tag_name** and **-tag_name**,
+   creating them if necessary.
+
+When evaluating the expression, the expression computation can roughly be expressed as: (amount of
+the move lines with `+` tag) - (amount of the move lines with `-` tag).
+
+'Aggregate Other Formulas' engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use this engine when you need to perform arithmetic operations on the amounts obtained for other
+expressions. Formula can be any expression with `+`, `-`, `/` and `*` operator. To refer to an
+expression, use the code of its **parent line** and its **label** (ex. **CODE.label**).
+
+**Sub-formulas** can be one of the following:
+
+- **if_above(CUR(amount))**: The value of the arithmetic expression will be returned only if it's
+  greater than the provided bound. Else, the result will be `0`;
+
+- **if_below(CUR(amount))**: The value of the arithmetic expression will be returned only if it's
+  lower than the provided bound. Else, the result will be `0`;
+
+- **if_between(CUR1(amount1), CUR2(amount2))**: The value of the arithmetic expression will be
+  returned only if it's strictly between the provided bounds. Else, it will be brought back to the
+  closest bound.
+
+`CUR` is the currency code in capital letters, and `amount` is the amount of the bound expressed in
+that currency.
+
+You can also use the `cross_report` sub-formula to match, if your expression refers to an expression
+defined in another report.
+
+'Prefix of Account Codes' engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This engine is used to match amounts made on accounts using the prefixes of these accounts' codes as
+variables in an arithmetic expression.
+
+.. example::
+   **21 + 10 - 5**
+     adds the balances of the move lines made on accounts whose codes start with `21` and `10`, and
+     subtract the balance of the ones on the accounts with the prefix `5`.
```

Comment:
```suggestion
  .. example::
   | **21 + 10 - 5**
   | Adds the balances of the move lines made on accounts whose codes start with `21` and `10`, and
     subtract the balance of the ones on the accounts with the prefix `5`.
```

Reply:
the indentation are intended to make it visually separate the formula from the text, and no capital to make it more understandable the text comes right after the formula, as they are connected

Reply:
I get what you mean, it looks like a 2-level list, but I'm still unsure that they look connected by doing so.
Here's my suggestion, but if you don't think it looks ok, I won't argue :)
```suggestion
.. example::
   **21 + 10 - 5**

   - adds the balances of the move lines made on accounts whose codes start with `21` and `10`, and
     subtract the balance of the ones on the accounts with the prefix `5`.
```

(it just adds a "point" before the sentence)